### DISCUSSION TITLE
How can I load subtitle for translation ?

### DISCUSSION THREAD

**Discussion by xidoc** [absadiki/subsai#59] [General]
Hi !
I have a subtitle(srt).
How can I load subtitle for translation on the UI or what's the command for translate ?

**Comment by absadiki** [comment]
Hi @xidoc,

You cannot load subtitles directly to the webui, but I have made a simple step by step colab notebook to achieve this.
You can find it here:  

* [Translation example](https://github.com/abdeladim-s/subsai/blob/main/examples/subsai_translation.ipynb): translate an already existing subtitles file. <a target="_blank" href="https://colab.research.google.com/github/abdeladim-s/subsai/blob/main/examples/subsai_translation.ipynb">
  <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
</a>

Hope this helps.
